docs-ref-conceptual/spring-framework/deploy-spring-boot-java-app-on-kubernetes.md

```diff
@@ -200,8 +156,34 @@ The `id` and `username` are the name of the registry. Use the `password` value f
    ```
    This command may take a while to complete.
 
-1. When you're using Azure Container Registry (ACR) with Azure Kubernetes Service (AKS), an authentication mechanism needs to be established. Follow the steps in [Authenticate with Azure Container Registry from Azure Kubernetes Service] to grant AKS access to ACR.
-
+1. When you're using Azure Container Registry (ACR) with Azure Kubernetes Service (AKS), you need to grant Azure Kubernetes Service pull access to Azure Container Registry. Azure creates a service principal when you are creating Azure Kubernetes Service to support cluster operability with other Azure resources. You can use this auto-generated service principal for authentication with an ACR registry, using the following scripts in PowerShell to grant the AKS-generated service principal pull access to an Azure container registry, please refer to
```

Comment:
You can use this auto-generated service principal for authentication with an ACR registry, using the following scripts in PowerShell to grant the AKS-generated service principal pull access to an Azure container registry, please ...
->
Please run the following scripts in PowerShell to grant the AKS-generated service principal pull access to an Azure Container Registry, see more details [Authenticate with Azure Container Registry from Azure Kubernetes Service].